### PR TITLE
Fixes issue with media not showing for activity if privacy is disabled

### PR DESCRIPTION
**Pull request by vaishaliagola27**
This PR fixes issue with comment media not showing on activity when privacy setting if off.

Related #1671

### CODE REVIEW

File: app/main/controllers/activity/RTMediaBuddyPressActivity.php [hunk 1]
```diff
@@ -621,17 +621,18 @@ public function bp_activity_posted_update( $content, $user_id, $activity_id ) {
 		$rtmedia_privacy = filter_input( INPUT_POST, 'rtmedia-privacy', FILTER_SANITIZE_NUMBER_INT );
 
 		if ( null !== $rtmedia_privacy ) {
-			$privacy = -1;
+
+			// Set Privacy 0 if media privacy is disabled.
+			$privacy = get_rtmedia_default_privacy();
```
Comment (abhijitrakas) [inline app/main/controllers/activity/RTMediaBuddyPressActivity.php:626]: @vaishaliagola27 - Is this tested with other scenarios like mention in https://github.com/rtCamp/rtMedia/pull/1415 issue?

Comment (vaishaliagola27) [reply app/main/controllers/activity/RTMediaBuddyPressActivity.php:626]: @abhijitrakas yes. I've tested all those scenarios.

**Comment by kapilpaul** [comment]
@pooja-muchandikar need to test this PR. please test this.

**Comment by pooja-muchandikar** [comment]
@kapilpaul 

Cross-checked with both scenarios privacy enabled and disabled and in both scenarios the commented media is visible.

**Pull request merged**